src/Tests.Microsoft.Security.Utilities.Core/SecretMaskerTests.cs

```diff
@@ -696,9 +696,10 @@ public void SecretMasker_NegativeMinimumSecretLengthSet()
 
         Assert.AreEqual("a***h", result);
     }
-    /*
```

Comment:
>/* [](http://example.com/codeflow?start=4&length=2)

Oops! In the original port, I left a few tests commented out with the result that I never authored a helper that ADO depends on. Now resolved.